src/backend/Csrs.Api/Repositories/RepositoryExtensions.cs

```diff
@@ -1,54 +1,54 @@
 ﻿using Csrs.Api.Authentication;
 using Csrs.Api.Configuration;
+using Csrs.Api.Repositories;
 using Simple.OData.Client;
 using System.Configuration;
 
-namespace Csrs.Api.Repositories
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Changed namespace so that in Program.cs, we do not require additional using statements and the `AddRepositories` method is just available.